homeassistant/components/airnow/sensor.py

```diff
@@ -69,19 +69,19 @@ class AirNowEntityDescription(SensorEntityDescription, AirNowEntityDescriptionMi
     ),
     AirNowEntityDescription(
         key=ATTR_API_PM25,
-        translation_key="pm25",
         icon="mdi:blur",
         native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
         state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PM25,
         value_fn=lambda data: data.get(ATTR_API_PM25),
         extra_state_attributes_fn=None,
     ),
     AirNowEntityDescription(
         key=ATTR_API_O3,
-        translation_key="o3",
         icon="mdi:blur",
         native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
         state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.OZONE,
```

Comment:
This sensor doesn't provide a valid unit of measurement for this device class. Only `µg/m³` is supported, `ppm` is not.